litmus-portal/frontend/src/views/ChaosWorkflows/BrowseWorkflow/TableData.tsx

```diff
@@ -74,13 +74,20 @@ const TableData: React.FC<TableDataProps> = ({ data, exeData }) => {
       </TableCell>
       <TableCell>
         <div className={classes.reliabiltyData}>
-          {exeData.phase === 'Failed' || exeData.phase === '' ? (
+          {exeData.finishedAt.length === 0 ? (
+            <Typography>
+              Overall RR: <span className={classes.warning}>60%</span>
+              <div className={classes.progressBar}>
+                <LinearProgressBar width={0.1} value={6} />
+              </div>
+            </Typography>
+          ) : exeData.phase === 'Failed' || exeData.phase === '' ? (
             <>
               <Typography>
                 Overall RR: <span className={classes.failed}>0%</span>
               </Typography>
               <div className={classes.progressBar}>
-                <LinearProgressBar width={2} value={0} />
+                <LinearProgressBar width={0.1} value={0} />
```

Comment:
are these benchmark values ? @amityt cc: @ksatchit we may calculate the resiliency score instead of showing the dummy/static value. Its a bug, if you are updating this please calculate based on weights. cc: @rajdas98